packages/atlas/src/hooks/useBlockTimeEstimation.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react'
+
+import { useJoystream } from '@/providers/joystream'
+
+const ESTIMATED_BLOCK_TIME = 6000
+
+export const useBlockTimeEstimation = () => {
+  const { joystream, proxyCallback } = useJoystream()
+
+  const [currentBlock, setCurrentBlock] = useState(0)
+  const [timeofTheLastBlock, setTimeOfTheLastBlock] = useState(0)
```

Comment:
Small typo here:
```suggestion
  const [timeOfTheLastblack, setTimeOfTheLastBlock] = useState(0)
```

Reply:
You also did a typo 😅  `timeOfTheLastblack`